### DISCUSSION TITLE
Audio Redirection

### DISCUSSION THREAD

**Discussion by TarLink48** [AlexandreRouma/SDRPlusPlus#648] [Q&A]
Hello again ! I wanted to ask if there is no possibility in SDR++ to redirect sound , for example , to a virtual cable without disabling the main sound ? How, for example , does the plugin for SDR# - MPX _Output ? Thanks ...
![armljr3lk8h81](https://user-images.githubusercontent.com/99340373/153729879-899c070a-f419-45f6-880e-b93dd28128e9.png)

**Comment by AlexandreRouma** [comment]
it's not, add a second demodulator.

**Comment by TarLink48** [comment]
It 's convenient and functional , but not for me unfortunately , I tried it ! I need a constant redirection of the sound of the active frequency , I am constantly moving through the ranges and I need a constant redirection of the entire sound ! How MPX Output works is the perfect solution for me , I just thought there was something similar for SDR++ ! Thank you for your time !
p.s. Here the person shows approximately what I need:
https://youtu.be/M7Luz7mhHI8

**Reply by AlexandreRouma** [reply]
Doing this would require completely rewriting the entire sink and stream system of SDR++. It's not meant at all to have several sinks for one stream/vfo.

**Comment by TarLink48** [comment]
I still don 't understand anything about it , it just seemed to me that the sound of SDR++ is better than Sharpe 's and decided to try it ! If there are no easy ways , then there is no need to invent anything , I will stay on the working scheme , thank you for the answer !
